### PR TITLE
Create Generate_reverse_templates.py

### PR DESCRIPTION
**Pull request by DTRademaker**
add_reverse_template_script

### CODE REVIEW

File: PANDORA/Database/Generate_reverse_templates.py [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+    change_chain_id(output_pdb, 'A', 'M')
+    change_chain_id(output_pdb, 'B', 'N')
+    change_chain_id(output_pdb, 'C', 'P')
+    
+    # Remove temporary files
+    if os.path.exists(temp_pdb):
+        os.remove(temp_pdb)
+
+if __name__ == "__main__":
+	reverse_pdb_structure('1J8H.pdb')
```
Comment (DarioMarzella) [inline PANDORA/Database/Generate_reverse_templates.py:526]: This should either be set to iterate over all the necessary templates and generate the full reversed database, or be a test that asserts if a file is generated properly, or be removed entirely.

File: PANDORA/Database/Generate_reverse_templates.py [hunk 2]
```diff
@@ -0,0 +1,526 @@
+import numpy as np
+from simtk.openmm.app import PDBFile, Modeller, ForceField, Simulation
+from pdbfixer import PDBFixer
+from simtk import openmm
+from openmm import LangevinIntegrator, Platform
+from openmm.unit import kelvin, picosecond, femtoseconds
+from Bio.PDB import PDBParser, PDBIO
+import os
+from typing import List, Dict, Tuple
+
```
Comment (DarioMarzella) [inline PANDORA/Database/Generate_reverse_templates.py:10]: As this is a stand-alone script that users are not necessarily supposed to use, it should maybe list in here or in a separate README the required packaged, package versions and so on, for the packages that are different from the ones normally installed by PANDORA.

**Review by DarioMarzella — CHANGES_REQUESTED** [review]
The code is nice, but it should be merged to a development branch first, not to master directly.

File: PANDORA/Database/Generate_reverse_templates.py (hunk 1 above)
Comment (DTRademaker) [reply PANDORA/Database/Generate_reverse_templates.py:526]: correct, thanks

File: PANDORA/Database/Generate_reverse_templates.py (hunk 2 above)
Comment (DTRademaker) [reply PANDORA/Database/Generate_reverse_templates.py:10]: will do, thx!

**Pull request closed without merging**